frontend/packages/pipelines-plugin/locales/en/pipelines-plugin.json

```diff
@@ -91,9 +91,7 @@
   "No task runs found": "No task runs found",
   "Download all task logs": "Download all task logs",
   "No logs found": "No logs found",
-  "PipelineRun logs": "PipelineRun logs",
   "Triggered by": "Triggered by",
-  "PipelineRun events": "PipelineRun events",
```

Comment:
This surprised me. We should check the pipelines pages that everything is still set correct.

Reply:
Before we had separate <Helmet> components for PipelineRun logs and PipelineRun events pages, now these are not required